src/snowflake/snowpark/_internal/analyzer/snowflake_plan.py

```diff
@@ -70,103 +74,23 @@
-                        col = match.group(1)
-                        children = [
-                            arg for arg in args if isinstance(arg, SnowflakePlan)
-                        ]
-                        remapped = [
-                            SnowflakePlan.Decorator.__wrap_exception_regex_sub.sub(
-                                "", val
-                            )
-                            for child in children
-                            for val in child.expr_to_alias.values()
-                        ]
-                        if col in remapped:
-                            unaliased_cols = (
-                                snowflake.snowpark.dataframe._get_unaliased(col)
-                            )
-                            orig_col_name = (
-                                unaliased_cols[0] if unaliased_cols else "<colname>"
-                            )
-                            ne = SnowparkClientExceptionMessages.SQL_PYTHON_REPORT_INVALID_ID(
-                                orig_col_name
-                            )
-                            raise ne.with_traceback(tb) from None
-                        elif (
-                            len(
-                                [
-                                    unaliased
-                                    for item in remapped
-                                    for unaliased in snowflake.snowpark.dataframe._get_unaliased(
-                                        item
-                                    )
-                                    if unaliased == col
-                                ]
-                            )
-                            > 1
-                        ):
-                            ne = SnowparkClientExceptionMessages.SQL_PYTHON_REPORT_JOIN_AMBIGUOUS(
-                                col, col
-                            )
-                            raise ne.with_traceback(tb) from None
-                        else:
-                            ne = SnowparkClientExceptionMessages.SQL_EXCEPTION_FROM_PROGRAMMING_ERROR(
-                                e
-                            )
-                            raise ne.with_traceback(tb) from None
-                    else:
-                        ne = SnowparkClientExceptionMessages.SQL_EXCEPTION_FROM_PROGRAMMING_ERROR(
-                            e
-                        )
-                        raise ne.with_traceback(tb) from None
-
-            return wrap
-
     def __init__(
-        self,
-        queries: List["Query"],
-        schema_query: str,
-        post_actions: Optional[List["Query"]] = None,
-        expr_to_alias: Optional[Dict[uuid.UUID, str]] = None,
-        session: Optional["snowflake.snowpark.session.Session"] = None,
-        source_plan: Optional[LogicalPlan] = None,
-        is_ddl_on_temp_object: bool = False,
-        api_calls: Optional[List[Dict]] = None,
+            self,
```

Comment:
Is this indent change intentional? I think our formatter should fix this?

Reply:
ok weird, I changed the indent back manually.